dpnp/backend/kernels/dpnp_krnl_common.cpp

```diff
@@ -81,8 +81,8 @@ cl::sycl::event dot(cl::sycl::queue& queue,
                     _DataType_output* result_out,
                     _DataType_input1* input1_in,
                     _DataType_input2* input2_in,
-                    size_t input1_strides,
-                    size_t input2_strides,
+                    ssize_t input1_strides,
```

Comment:
`ssize_t` is capable to store values from -1 to max of `size_t`. Are you sure you won't have values less than -1?